qipr_approver/approver/templates/templatetags/form_input.html

```diff
@@ -1,4 +1,4 @@
 <div class="form-input-wrapper input_{{ name }} ">
     <label for="{{ name }}">{{ label }}</label>
-    <input class="{{ input_class_list }}" name="{{ name }}" type="{{ type }}" value="{{ value }}" {{ is_disabled }}/>
+    <input class="{{ input_class_list }}" name="{{ name }}" type="{{ type }}" value="{{ value }}" placeholder="{{ placeholder }}" {{ is_disabled }}/>
```

Comment:
The other dictionaries (like user.title from the about_you_form.py file) seem to work fine ...in chrome at least. Is it bad code design to request a variable that might not exist? 

Another option: First, we could add another parameter to the form_input function so it becomes:
form_input(input_dict, is_disabled=False, readonly=False)

Second, we have the about_you.html pass the extra parameter in the user_name template tag call, like we do with the is_disabled parameters for {% form_input form.last_login True %}